flybywiresim/aircraft
# fix(fuel): Lowered C/D spawn fuel, fixed center tank refueling

Fixes #8092

## Summary of Changes


This PR aims to lower the fuel amount on C/D spawns to 2.5 tons. This way it reflects reality a bit better and almost always requires to refuel before flight.

Additionally I also changed the logic behind loading the saved fuel from last shutdown. Previously it would always be loaded no matter where one spawns. To me that didn't make sense as If someone wants to spawn on the runway or in the air, they are obviously forgoing refueling and therefore need a reasonable amount of fuel in the tanks. Currently that is 50% of the fuel load. 

This can be changed by adding tank levels to the FLT files if desired.

Finally fixed the longstanding issue where our plane would only fuel the center tanks after the wing tanks had been filled. In reality both center and wing tanks get filled at the same time.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):\_chaoz_

## Testing instructions


First make sure you are using the default FBW livery and that the A32NX.ini file in the location described here is deleted:
https://docs.flybywiresim.com/fbw-a32nx/feature-guides/loading-fuel-weight/?h=aircraftstate#fuel-persistence

1. Spawn C/D
2. Check fuel amount, should be 2.5 tons in only the inner tanks
3. Fill up fuel tanks completely and observe center tanks getting filled while wing tanks are filling
4. Detank fuel and observe the same in reverse
5. Spawn on runway
6. Check fuel amount to be 50% 



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
